Update README: Include aarch64 windows and full build info (#700)
cargo-bins/cargo-binstall: README.md

| macos   | m1      | https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-aarch64-apple-darwin.zip |
| macos   | universal | https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-universal-apple-darwin.zip |
| windows | x86\_64 | https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-x86_64-pc-windows-msvc.zip |
| windows | arm64 | https://github.com/cargo-bins/cargo-binstall/releases/latest/download/cargo-binstall-aarch64-pc-windows-msvc.zip |

We also provide pre-built artifacts with debuginfo for Linux and Mac.

> These artifacts is postfixed with `.full.tgz` on Linux and `.full.zip` on Mac.

Reviewer: ```suggestion
These artifacts are suffixed with `.full.tgz` on Linux and `.full.zip` on Mac and Windows.
```
